Frontend/Gmfctn/src/app/modules/layout/layout-routing.module.ts

```diff
@@ -0,0 +1,33 @@
+import { NgModule } from '@angular/core';
+import { RouterModule, Routes } from '@angular/router';
+import { LayoutComponent } from './layout.component';
+
+const routes: Routes = [
```

Comment:
This routes should be on the app level as the layout itself is not about the app logic, but about the main app look

Reply:
Fixed